### DISCUSSION TITLE
blog/iakfelvgul5yfy9t

### DISCUSSION THREAD

**Discussion by giscus** [CNSeniorious000/yuque-blog#30] [giscus]
# blog/iakfelvgul5yfy9t

最后更新于：12月28日

https://muspimerol.site/blog/iakfelvgul5yfy9t

**Comment by hexWars** [comment]
ps: 评论可以开在单独的一个仓库，我之前是开在博客仓库的但是我私有之后就无法使用了

**Reply by CNSeniorious000** [reply]
合理。你是不是就得一个个 transfer 过去哈哈哈哈
不过为什么你博客仓库要设为私有呢

**Reply by hexWars** [reply]
我直接没有transfer，那个评论和用户绑定了转移了也都是我的名字而不是原来用户的

至于为啥，删掉了一些个人信息是1，如果别人clone下来看记录我不得***，设成私有可以随便改是2，无需顾忌太多东西

**Reply by CNSeniorious000** [reply]
哈哈哈哈 像我这种实名上网人士就没这种顾虑了

![image](https://github.com/user-attachments/assets/5e70fa53-1260-41cf-ac6a-9c5b3c2ed012)

GitHub Discussion 的这个 Transfer 应该不会改变什么吧
我看 giscus 文档里也说如果之前用的 utterances 之类的工具也可以直接[迁移](https://github.com/giscus/giscus/blob/main/README.zh-CN.md#%E8%BF%81%E7%A7%BB)数据过来的：

**Reply by hexWars** [reply]
我说的 Transfer 是一个个手动转移QWQ，我试了一下还是不行，不过原来的评论就不多也没什么

感觉开源代码还是需要一点美感的，我的库会有各种实验性的东西，也比较难看
